### PR TITLE
Add more release qualification tests

### PR DESCRIPTION
**Pull request by chxchx**
Job scripts can be found in https://github.com/istio-releases/daily-release/tree/master/prow

**Release note**:

```release-note
none
```

### CODE REVIEW

**Review by sebastienvas — APPROVED** [review]

File: prow/config.yaml [hunk 1]
```diff
@@ -1101,6 +1101,188 @@ presubmits:
...
+          secretName: service-account
+      - name: daily-release-e2e-rbac-kubeconfig
+        secret:
+          secretName: daily-release-e2e-rbac-kubeconfig
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+  - name: daily-e2e-cluster_wide-auth
+    agent: kubernetes
+    spec:
```
Comment (yutongz) [inline prow/config.yaml]: I think we need to add "always_run: true" here. This is part of presubmit.
Also "context" which will show on github and "branch" master

File: prow/config.yaml [hunk 2]
```diff
@@ -1101,6 +1101,188 @@ presubmits:
...
+      volumes:
+      - name: service
+        secret:
+          secretName: service-account
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+  - name: daily-e2e-cluster_wide-auth-default
+    agent: kubernetes
+    spec:
```
Comment (yutongz) [inline prow/config.yaml]: Same here.

File: prow/config.yaml (hunk 1 above)
Comment (yutongz) [reply prow/config.yaml]: And also add "rerun_command" and "trigger" like jobs before, although we probably won't use them in release qualification. Just for consistency.

**Comment by chxchx** [comment]
PTAL

**Pull request merged**